pandas-stubs/core/series.pyi

```diff
@@ -1623,6 +1628,11 @@ class Series(IndexOpsMixin[S1], NDFrame):
         self, other: int | np_ndarray_anyint | Series[int]
     ) -> Series[int]: ...
     # def __array__(self, dtype: Optional[_bool] = ...) -> _np_ndarray
+    @overload
+    def __div__(self: Series[int], other: Series[int]) -> Series[float]: ...
+    @overload
+    def __div__(self: Series[int], other: int) -> Series[float]: ...
+    @overload
     def __div__(self, other: num | _ListLike | Series[S1]) -> Series[S1]: ...
```

Comment:
We need to remove `__div__()`.  It is python 2.  In python 3, we should be using `__truediv__` .  Same goes for removing `__rdiv__` and using `__rtruediv__` instead.  So if you can migrate the overloads accordingly, I think this will get fixed.

I just learned that by looking up the issue!